### DISCUSSION TITLE
Unable to push due to missing ref??

### DISCUSSION THREAD

**Discussion by sunnivaliu** [community/community#57579] [Other Feature Feedback, Questions, & Ideas]
### Bug on git push

Hi,

I keep getting this error when I've just opened a **new repository and push my Unity project** on to that using my github desktop

`Uploading LFS objects:  99% (339/343), 20 MB | 0 B/s, done.
Git LFS upload missing objects:
  (missing) memoryAug-unity/Library/PackageCache/com.htc.upm.vive.openxr@1.0.10/OpenXRSceneUnderstanding/Samples~/MeshingFeaturePlugin/universalwindows/x64/MeshingFeaturePlugin.dll (95c16fcc2b23429dedb1452bf37e2a97910c2d506a9b385093683ae2c918de42)
  (missing) memoryAug-unity/Library/PackageCache/com.htc.upm.vive.openxr@1.0.10/OpenXRSceneUnderstanding/Samples~/MeshingFeaturePlugin/universalwindows/arm64/MeshingFeaturePlugin.dll (cc62cc0759279e0eee751ce13008d40b5a8f246d946b0fa418f61e57b8aceae9)
  (missing) memoryAug-unity/Library/PackageCache/com.htc.upm.vive.openxr@1.0.10/OpenXRSceneUnderstanding/Samples~/MeshingFeaturePlugin/universalwindows/arm32/MeshingFeaturePlugin.dll (86855ad1c4b7c3007c2a526ddd4c4df8514d63272c2425142f3a1dba4c1627a9)
  (missing) memoryAug-unity/Library/PackageCache/com.htc.upm.vive.openxr@1.0.10/OpenXRSceneUnderstanding/Samples~/MeshingFeaturePlugin/android/arm64/libMeshingFeaturePlugin.so (cd6b3a267584526517d8dcdc07078102b018b2b27efff637f7b1fb0730406252)
Enumerating objects: 21780, done.
Counting objects: 100% (21780/21780), done.
Delta compression using up to 16 threads
Compressing objects: 100% (10147/10147), done.
Writing objects: 100% (21779/21779), 325.17 MiB | 9.63 MiB/s, done.
Total 21779 (delta 12197), reused 20480 (delta 11596), pack-reused 0
remote: Resolving deltas: 100% (12197/12197), done.        
remote: warning: See https://gh.io/lfs for more information.        
remote: warning: File memoryAug-unity/Library/ArtifactDB is 64.00 MB; this is larger than GitHub's recommended maximum file size of 50.00 MB        
remote: warning: GH001: Large files detected. You may want to try Git Large File Storage - https://git-lfs.github.com.        
remote: error: GH008: Your push referenced at least 4 unknown Git LFS objects:        
remote:     cd6b3a267584526517d8dcdc07078102b018b2b27efff637f7b1fb0730406252        
remote:     86855ad1c4b7c3007c2a526ddd4c4df8514d63272c2425142f3a1dba4c1627a9        
remote:     cc62cc0759279e0eee751ce13008d40b5a8f246d946b0fa418f61e57b8aceae9        
remote:     ...        
remote: Try to push them with 'git lfs push --all'.        
To https://github.com/Augmented-Perception-Lab/memory-augmentation.git
 ! [remote rejected]   prototype -> prototype (pre-receive hook declined)
error: failed to push some refs to 'https://github.com/Augmented-Perception-Lab/memory-augmentation.git'`

**I've tried git push -f origin, but unfortunately doesn't work...
Does anyone know how to resolve this?**

Thanks a lot!

**Comment by mwt** [comment]
If you haven't tried yet, you can `git lfs push --all`
